framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1267,6 +1263,28 @@ private boolean shouldSuppressWarnings(Tree tree, String err) {
     }
 
     /**
+     * Should unchecked code defaults be used for the kind of code indicated by the parameter
+     * @param kindOfCode source or bytecode
+     * @return whether unchecked code defaults should be used
+     */
+    public boolean useUncheckedCodeDefault(String kindOfCode) {
+        String option = this.getOption("useDefaultsForUncheckedCode", "-source,-bytecode");
+        if(option == null){
```

Comment:
If the user only provides a setting for one option, the other won't be defaulted.